examples/protocol_specification_ex/ml_trade.yaml

```diff
@@ -0,0 +1,20 @@
+name: ml_trade
+author: fetchai
+version: 0.1.0
+license: Apache 2.0
+description: 'A protocol for trading data for training and prediction purposes.'
+speech_acts:
+  cfp:
+    query: Query
+  terms:
+    terms: Descrition
+  accept:
+    terms: Descrition
+    tx_digest: str
+  data:
+    terms: Descrition
+    data: Tuple
```

Comment:
I think it needs to be strongly typed, like `Tuple[Array, Array]` or whatever you want to have there...

Reply:
Done